### PR TITLE
[9.0] document_page_quality_manual

### PR DESCRIPTION
**Pull request by naousse**

### CODE REVIEW

File: document_page_quality_manual/README.rst [hunk 1]
```diff
@@ -1,3 +1,6 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline document_page_quality_manual/README.rst:1]: The README is incomplete

Comment (max3903) [inline document_page_quality_manual/README.rst:1]: @elicoidal Do you mean anything else in addition to the target of the badge link?

Comment (elicoidal) [inline document_page_quality_manual/README.rst:1]: Please check the links at bottom (https x2)
Rest looks good

File: document_page_quality_manual/__openerp__.py [hunk 2]
```diff
@@ -21,33 +21,20 @@
 
 {
     'name': 'Quality Manual',
-    'version': '8.0.1.0.0',
+    'version': '9.0.1.0.0',
     'category': 'Management System',
     'complexity': "easy",
```
Comment (elicoidal) [inline document_page_quality_manual/__openerp__.py]: remove

File: document_page_quality_manual/__openerp__.py [hunk 3]
```diff
@@ -21,33 +21,20 @@
...
     'license': 'AGPL-3',
     'depends': [
-        'mgmtsystem_manuals'
+        'mgmtsystem_manual'
     ],
     'data': [
-        'data/document_page_data.xml'
+        'data/document_page_quality_manual.xml'
     ],
     'demo': [],
```
Comment (elicoidal) [inline document_page_quality_manual/__openerp__.py]: remove

File: document_page_quality_manual/README.rst (hunk 1 above)
Comment (max3903) [inline document_page_quality_manual/README.rst:1]: @elicoidal The template has https links as well:
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

Comment (elicoidal) [inline document_page_quality_manual/README.rst:1]: bottom links should https://odoo-community.org

**Comment by dreispt** [comment]
@naousse ping!

**Comment by max3903** [comment]
:+1:

File: document_page_quality_manual/README.rst [hunk 4]
```diff
@@ -1,29 +1,39 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License
+
+==============
 Quality Manual
 ==============
 
-This module was written to provide a quality manual template. The template has the same structure as the ISO 9001 standard.
+This module provide a quality manual template. The template has the same structure as the ISO 9001 standard.
```
Comment (dreispt) [inline document_page_quality_manual/README.rst:9]: s/provide/provides

**Comment by dreispt** [comment]
:+1:

**Pull request merged**